nanoFramework.Device.Bluetooth/BluetoothNanoDevice.cs

```diff
@@ -31,16 +31,18 @@ static BluetoothNanoDevice()
 
         internal static string DeviceName { get => _deviceName; set => _deviceName = value; }
 
+        /// <summary>
+        /// Check if current mode otherwise switch to mode
+        /// </summary>
+        /// <param name="expectedMode"></param>
+        /// <exception cref="InvalidOperationException"></exception>
```

Comment:
Please add a description here :-) But on what I can see, it doesn't seems to raise any exception anymore

Reply:
Exception comes from native code.

Reply:
Comment updated